cuentociacua.txt

```diff
@@ -0,0 +1 @@
+Esta es la historia de dos niñas mimadas, prepotentes y fastidiosas que solo les gustaba hablar sobre lo que las otras niñas jugaban. Tanto lo hicieron que perdieron la alegriade vivir.
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.